.github/workflows/deploy-xtgeo-pypi.yml

```diff
@@ -4,83 +4,164 @@ on:
-    git clone --depth 1 https://github.com/equinor/xtgeo-testdata ../xtgeo-testdata
-    pytest tests/test_surface --disable-warnings
-  CIBW_TEST_COMMAND_WINDOWS: |
-    pushd {project}
-    rm -rf ../xtgeo-testdata
-    git clone --depth 1 https://github.com/equinor/xtgeo-testdata ../xtgeo-testdata
-    pytest tests/test_surface --disable-warnings
-  CIBW_TEST_COMMAND_LINUX: |
-    pushd {project}
-    rm -rf ../xtgeo-testdata
-    git clone --depth 1 https://github.com/equinor/xtgeo-testdata ../xtgeo-testdata
-    pytest tests --disable-warnings
-
-  CIBW_BUILD: |
-    cp36-manylinux_x86_64
-    cp37-manylinux_x86_64
-    cp38-manylinux_x86_64
-    cp39-manylinux_x86_64
-    cp36-macosx_x86_64
-    cp37-macosx_x86_64
-    cp38-macosx_x86_64
-    cp39-macosx_x86_64
-    cp36-win_amd64
-    cp37-win_amd64
-    cp38-win_amd64
-    cp38-win_amd64
-    cp39-win_amd64
-
+# Verify that wheel build jobs succeed
 jobs:
-  build_wheels:
-    name: Deploy wheels to PYPI for ${{ matrix.os }}
+  deploy_linux_cibuildwheel:
+    name: CIBW deploy ${{ matrix.cibw_python }} on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+
+    env:
+      CIBW_BEFORE_ALL: "sh scripts/swig_manylinux.sh"
+      CIBW_BEFORE_TEST: >
+        pushd {project} &&
+        pip install -r requirements/requirements.txt &&
+        pip install -r requirements/requirements_test.txt &&
+        git clone --depth 1 https://github.com/equinor/xtgeo-testdata ../xtgeo-testdata
+      CIBW_TEST_COMMAND: >
+        pushd {project} &&
+        pytest tests --disable-warnings
+      CIBW_BUILD: ${{ matrix.cibw_python }}-manylinux_x86_64
+      CIBW_MANYLINUX_X86_64_IMAGE: manylinux2010
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        cibw_python: [cp36, cp37, cp38, cp39]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+
+      - name: Install cibuildwheel
+        run: >
+          python -m pip install pip -U &&
+          python -m pip install wheel &&
+          python -m pip install cibuildwheel
+
+      - name: Build wheels and test
+        run: |
+          python -m cibuildwheel --output-dir wheelhouse
+
+      - name: Publish to PYPI Linux
+        env:
+          TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+          TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+        run: >
+          pip install twine &&
+          twine upload wheelhouse/*
 
+  deploy_macos_cibuildwheel:
+    name: CIBW deploy ${{ matrix.cibw_python }} on ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
 
+    env:
+      CIBW_BEFORE_ALL: "brew install swig"
+      CIBW_BEFORE_TEST: >
+        pushd {project} &&
+        pip install -r requirements/requirements.txt &&
+        pip install -r requirements/requirements_test.txt &&
+        git clone --depth 1 https://github.com/equinor/xtgeo-testdata ../xtgeo-testdata
+      CIBW_TEST_COMMAND: >
+        pushd {project} &&
+        pytest tests --disable-warnings &&
```

Comment:
Just to make sure, you were not intending to add more here? Thinking about the trailing `&&`

Reply:
Correct, typo; is fixed